addons/l10n_es/data/account_chart_template_account_account_link.xml

```diff
@@ -15,6 +15,9 @@
             <field name="default_cash_difference_income_account_id" ref="account_common_778"/>
             <field name="default_cash_difference_expense_account_id" ref="account_common_678"/>
             <field name="default_pos_receivable_account_id" ref="account_common_4301" />
+            <field name="account_journal_suspense_account_id" ref="account_common_572991" />
+            <field name="account_journal_payment_debit_account_id" ref="account_common_572992" />
```

Comment:
This one should be 4312, changing the type to current assets.

Reply:
Of course